### PR TITLE
Create webrtc voice api.md

### PR DESCRIPTION
**Pull request by sushanthmavenir**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

* documentation for the webrtc voice api

#### Special notes for reviewers:

This is an initial draft version, additional changes will be required based on the latest yaml changes.

### CODE REVIEW

File: documentation/API_documentation/webrtc voice api.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Outgoing calls made through the service will display the user's MSISDN as the caller number, ensuring that the recipient recognizes the user's phone number on their caller ID.
+
+Incoming calls introduce a unique functionality whereby multiple devices will ring simultaneously. This includes the user's smartphone and the web endpoint they have registered from. This design enables users to conveniently answer calls from any of these devices, depending on their preference or accessibility. The service is compatible with various devices that support web browsing environments, such as laptops, PCs, and XR immersive glasses.
+
+The service specification emphasises the accessibility and flexibility of utilising one's phone number across different non-SIM card devices. By leveraging this web-based service, users can extend their communication capabilities beyond traditional SIM-based devices, enhancing their overall communication experience.
+
+## Introduction
+
+This service brings new functionalities for the WebRTC services whose idea is to integrate BYON API to maintain the client number in out-bound and in-bound calls from web comms service.
```
Comment (pradeepachar-mavenir) [inline documentation/API_documentation/webrtc voice api.md:13]: comms to be changed to communications

File: documentation/API_documentation/webrtc voice api.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Roles: Customer-user
+
+Actors: Application service providers, hyperscalers, application developers.
+
+Scope:
+
+1. Register to Network – Register the client endpoint instance to MNO's IMS Network.
+2. Create Notification Channel – Create Notification channel of specified type to receive notifications from the network (e.g. incoming call).
+3. Place an Out Call – Place an outbound voice call request to the MNO's IMS Network.
```
Comment (pradeepachar-mavenir) [inline documentation/API_documentation/webrtc voice api.md:39]: We might have to include receive calls as well.

File: documentation/API_documentation/webrtc voice api.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+### Roles, Actors and Scope
+
+Roles: Customer-user
+
+Actors: Application service providers, hyperscalers, application developers.
+
+Scope:
+
+1. Register to Network – Register the client endpoint instance to MNO's IMS Network.
+2. Create Notification Channel – Create Notification channel of specified type to receive notifications from the network (e.g. incoming call).
```
Comment (pradeepachar-mavenir) [inline documentation/API_documentation/webrtc voice api.md:38]: We might have to include independent notification channel provider.

Comment (sushanthmavenir) [reply documentation/API_documentation/webrtc voice api.md:38]: The API is independent of whether it is implemented by independent channel provider or WebRTC gateway.  the API definition will remain the same, rest is implementation dependent.

**Comment by sushanthmavenir** [comment]
@TEF-RicardoSerr can you please complete this review so that i can merge it into mainline?

**Review by TEF-RicardoSerr — APPROVED** [review]
Nothing to add for now. Thank you!

**Pull request merged**